Makefile

```diff
@@ -5,8 +5,8 @@ DOCKER_ORG=interconnectedcloud
 PWD=$(shell pwd)
 
  # This is the latest version of the Qpid Dispatch Router
-DISPATCH_VERSION=1.15.0
-PROTON_VERSION=0.33.0
+DISPATCH_VERSION=1.16.0-freeze
```

Comment:
This won't work for the url below.

Reply:
Yes, I was testing consuming it from github and it worked, but not working from apache,
I will research a bit more and I will try not to force tags to be pushed into this repo initially.